src/cpp/lib/QtWidgets/QTreeWidget/qtreewidget_wrap.cpp

```diff
@@ -74,3 +79,83 @@ Napi::Value QTreeWidgetWrap::selectedItems(const Napi::CallbackInfo& info) {
 
   return napiItems;
 }
+
+Napi::Value QTreeWidgetWrap::setColumnCount(const Napi::CallbackInfo& info) {
+  Napi::Env env = info.Env();
+  Napi::HandleScope scope(env);
+
+  int columns = info[0].As<Napi::Number>().Int32Value();
+  this->instance->setColumnCount(columns);
+
+  return Napi::Value();
```

Comment:
knitpick: I would recommend returning `env.Null();` since that is what we are following in rest of the code base.

Reply:
I agree.